compose/compose.go

```diff
@@ -248,6 +249,18 @@ func getJaeger() service {
 	return svc
 }
 
+func getRatel() service {
+	svc := service{
+		Image:         "dgraph/dgraph:" + opts.Tag,
+		ContainerName: "ratel",
+		Ports: []string{
+			toExposedPort(8000),
```

Comment:
I'll add a Ratel port flag. Another thing we can do is have a flag to only expose the posts and not set the direct host:port mapping so that Docker picks a random port to expose to the host.

Reply:
Added a `--ratel_port` option to make the port configurable.